serverless.yml

```diff
@@ -51,6 +51,20 @@ provider:
         - s3:PutBucketAcl
       Resource: "*"
 
+    - Effect: Allow
+      Action:
+        - sns:AddPermission
```

Comment:
I don't think these permissions (sns:* ) are required. Or I'm missing sth ? 
Please consider [The principle of least privilege](https://en.wikipedia.org/wiki/Principle_of_least_privilege) (PoLP)